LS-446-remove-replicated (#33)
datastax/pulsar-docs: modules/ROOT/pages/index.adoc

> In addition to the distribution of https://pulsar.apache.org/en/versions/[Apache Pulsar {pulsar-version}], DataStax Luna Streaming provides:

Reviewer: Do we want to cover other components of the DS LS?
* AdminConsole
* HeartBeat
* Burnell
* etc.

If so, these components have the biggest difference regarding deployment between K8s and tarball

* A helm chart that can deploy and manage Pulsar on your current CNCF conformant Kubernetes infrastructure

> * Cassandra, Elastic, Kinesis, Kafka, and JDBC connectors

Reviewer: This not a full list. Are we planning to provide a connenctor oriented document space? If so, we can add a link here.

* Transparent batching of messages

== What's next?

* If you have an existing Kubernetes environment, deploy Luna Streaming with a xref::quickstart-helm-installs.adoc[Helm Installation].
* If you have a bare metal or a cloud environment, see xref::quickstart-server-installs.adoc[Server/VM Installation].
* If you want to learn about monitoring with Pulsar Heartbeat, see xref::pulsar-monitor.adoc[Pulsar Heartbeat].

> If you have questions about Luna Streaming, see xref::faqs.adoc[Luna Streaming FAQs].

Reviewer 2: ```suggestion
* If you have questions about Luna Streaming, see xref::faqs.adoc[Luna Streaming FAQs].
```

== Where to go next

> If you've got an existing Kubernetes environment:: To deploy Luna Streaming in your existing Kubernetes environment, see xref::quickstart-helm-installs.adoc[Helm Installation].
> 

Reviewer 2: ```suggestion
* If you have an existing Kubernetes environment, deploy Luna Streaming with a xref::quickstart-helm-installs.adoc[Helm Installation].
```

> If you've got bare metal or a cloud environment:: To deploy Luna Streaming environment from scratch on bare metal or in a cloud environment, see xref::quickstart-server-installs.adoc[Server/VM Installation].
> 

Reviewer 2: ```suggestion
* If you have a bare metal or a cloud environment, see xref::quickstart-server-installs.adoc[Server/VM Installation].
```

> If you'd like to learn about Pulsar Heartbeat:: See xref::pulsar-monitor.adoc[Pulsar Heartbeat].
> 

Reviewer 2: ```suggestion
* If you want to learn about monitoring with Pulsar Heartbeat, see xref::pulsar-monitor.adoc[Pulsar Heartbeat].
```

> If you have more questions...:: For additional information about Luna Streaming, see xref::faqs.adoc[Luna Streaming FAQs].

Reviewer 2: ```suggestion
If you have questions about Luna Streaming, see xref::faqs.adoc[Luna Streaming FAQs].
```
